### PR TITLE
Update copyright note in documentation

### LINKED ISSUE #231: Copyright notice out of date
In the [documentation](https://pyresample.readthedocs.io/en/stable/index.html), the copyright notice reads:

> © Copyright 2013, Esben S. Nielsen

That would appear to be out of date.

### PR DESCRIPTION
**Pull request by gerritholl**
Update the copyright notice in the readthedocs documentation.

 - [x] Closes #231 
 - [x] Passes ``git diff origin/main **/*py | flake8 --diff``

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 import os
 import sys
+from datetime import datetime
```
[the remark this answers is not in the document]
Comment (gerritholl) [reply docs/source/conf.py:17]: You're wrong!  It's used in line 73.

File: docs/source/conf.py [hunk 2]
```diff
@@ -69,7 +70,7 @@
 
 # General information about the project.
 project = u'pyresample'
-copyright = u'2013, Esben S. Nielsen'
+copyright = "2013-{datetime.utcnow():%Y}, the Pytroll team"
```
Comment (djhoese) [inline docs/source/conf.py:73]: Don't you need an `f` to make this an f-string...and only then would stickler be incorrect.

Comment (djhoese) [reply docs/source/conf.py:73]: Also I think in our license headers on files we've stuck with "pyresample developers". I don't feel strongly either way what this copyright says but maybe the consistency would be nice.

Comment (djhoese) [reply docs/source/conf.py:73]: I changed my mind, I do prefer the consistency.

```suggestion
copyright = f"2013-{datetime.utcnow():%Y}, Pyresample developers"
```

**Pull request merged**